includes/class-payment-information.php

```diff
@@ -237,14 +237,8 @@ public static function get_payment_method_from_request( array $request ): string
 	 * @return \WC_Payment_Token|NULL
 	 */
 	public static function get_token_from_request( array $request ) {
-		$payment_method = $request['payment_method'] ?? null;
-		if ( Sepa_Payment_Gateway::GATEWAY_ID === $payment_method ) {
-			$request_gateway_id = Sepa_Payment_Gateway::GATEWAY_ID;
-		} else {
-			$request_gateway_id = CC_Payment_Gateway::GATEWAY_ID;
-		}
-
-		$token_request_key = 'wc-' . $request_gateway_id . '-payment-token';
+		$payment_method    = $request['payment_method'] ?? null;
```

Comment:
Since the previous conditional was checking the equality of `payment_method` and `GATEWAY_ID` and then just using the `GATEWAY_ID`, I thought this would be more versatile as we add new payment methods rather than having several conditionals for each method to only assign the variable the `payment_method` value anyway. Since `null` resolves to empty string during concatenation, I thought this process was safe and the `isset` check will kick us out.